src/e2e-test/features/salesforcebatchsource/RunTime.feature

```diff
@@ -27,7 +27,7 @@ Feature: Salesforce Batch Source - Run time Scenarios
     And Navigate to the properties page of plugin: "Salesforce"
     And fill Authentication properties for Salesforce Admin user
     And Enter input plugin property: "referenceName" with value: "Reference"
-    And Enter input plugin property: "sObjectName" with value: "sobject.lead"
+    And Enter input plugin property: "sObjectName" with value: "automation"
```

Comment:
Instead of simply writing ```automation```  write ```sobject.Automation_custom_c``` for a better readability

Reply:
done